### ISSUE TITLE
Configure DTL timeouts for a volumedriver

### ISSUE THREAD

**Issue by redlicha** [openvstorage/framework#1234]
https://github.com/openvstorage/volumedriver/issues/131
https://github.com/openvstorage/volumedriver/pull/132

Proposed default: 1 second ("out of thin air", better informed suggestions / discussion desired!)

**Comment by khenderick** [comment]
I don't like to update some volumedriver default value (that's apparently not good for every environment out there) to a framework default value (that won't be good for every environment out there either). It's just adding work that doesn't add value.

Imo, the only reason for the framework to overwrite a volumedriver default value, should be when the framework can calculate a sane value based on the envrionment, or asks the customer for what the value should be (in case it can't calculate a better value at that time). If it's just adding another hardcoded value, we can as well change it in the volumedriver as well.

**Comment by redlicha** [comment]
I agree. Having the fwk provide a value based on information it has, e.g. DTL type (disk vs. mem based, RDMA vs. TCP) was the motivation for this ticket.

**Comment by wimpers** [comment]
https://github.com/openvstorage/volumedriver/issues/194 changed the default on voldrv
